specification/protocol/generate_rest.yaml

```diff
@@ -0,0 +1,186 @@
+openapi: 3.1.0
+info:
+  title: Open Inference API for text generation
+  description: Open Inference API for text generation
+  version: 1.0.0
+components:
+  schemas:
+    GenerateRequest:
+      type: object
+      required: 
+        - text_input
+      properties:
+        text_input:
+          type: string
+        parameters:
+          $ref: '#/components/schemas/GenerateParameters'
+    GenerateParameters:
+        allOf:
+          $ref: '#/components/schemas/GenerateParameters'
+        type: object
+        additionalProperties: {}
+        properties:
+          temperature:
+            type: number
+            format: float
+            default: null
```

Comment:
vllm defaults to 1
https://github.com/vllm-project/vllm/blob/main/vllm/sampling_params.py#L100

Reply:
Updated